src/components/filters/FiltersInfos.js

```diff
@@ -9,24 +10,24 @@ const FiltersInfos = ({
 
   const getValue = (key) => {
     const filterValue = filters[key].value
+    const valuesArray = [].concat(filterValue)
 
     switch(description[key].type) {
-      case 'SELECT':
-        const valuesArray = [].concat(filterValue)
+      case FILTER_TYPE.SELECT:
         const labels = valuesArray.map((val) => {
           const option = description[key].options.find(option => option.value === val)
           return option.label
         })
         return labels.join(', ')
-        break;
-      case 'RANGE':
-        if (filterValue.min || filterValue.max)
-          return `${filterValue.min} to ${filterValue.max} `
-        else
-          throw new Error('MIN and MAX values not defined for Filter range')
-        break;
+      case FILTER_TYPE.RANGE:
+        let value = ""
+        if (filterValue.min !== undefined) {value += ` min: ${filterValue.min}`}
+        if (filterValue.max !== undefined) {value +=` max: ${filterValue.max}`}
```

Comment:
Lint:
 - Missing space after equal.
 - If are using braces for a single statement, but other single-statement ifs dont use braces (such as the one right below)

Also the space before ` min` and ` max` isn't ideal. I would at least put it after so it's not visible.
Also I would have implemented it such that only `min = 42` would render as `more than 42`, `max = 42` would render as `less than 42`, and `min = 0, max = 10` would render as `from 0 to 10` or `between 0 and 10`.